rfc/src/rfcs/0012-list.md

```diff
@@ -132,6 +152,30 @@ For example:
 }
 ```
 
+#### Output Schema
+The formal output schema is:
+```
+Kani Version: float
+File Version: int
+Standard Harnesses: List[Object]
+    - file: string
+    - harnesses: List[string]
+Contract Harnesses: List[Object]
+    - file: string
+    - harnesses: List[string]
+Contracts: List[Object]
+    - function: string
+    - file: string
+    - harnesses: List[string]
+Totals: Object
+    - standard-harnesses: int
+    - contract-harnesses: int
+    - functions-with-contracts: int
+    - contracts: int
+```
+
+All sections will be present in the output, regardless of the result.
```

Comment:
What happens if there are no contracts, do we just present an empty list?

Reply:
Yes, the output schema just has a `List` (without None) since the list can just be empty. The example has a case of this. Can add text to make that explicit if it's unclear.

Reply:
Yes, would be better to add to the RFC to make it explicit